pdc/settings.py

```diff
@@ -93,7 +93,7 @@
     'DEFAULT_METADATA_CLASS': 'contrib.bulk_operations.metadata.BulkMetadata',
 
     'DEFAULT_FILTER_BACKENDS': ('rest_framework.filters.DjangoFilterBackend',
-                                'rest_framework.filters.OrderingFilter'),
+                                'pdc.apps.utils.utils.RelatedNestedOrderingFilter'),
```

Comment:
Yeah. @bliuredhat is right. we need to update the local dist file as well, so that the user can use local.py correctly